pkg/config/config.go

```diff
@@ -1108,6 +1108,10 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("container_image.enabled", false)
 	bindEnvAndSetLogsConfigKeys(config, "container_image.")
 
+	// Remote process collector
+	config.BindEnvAndSetDefault("workloadmeta.remote_process_collector.enabled", false)
```

Comment:
If you use `Features`, you don't need this, you can activate the feature with `autoconfig_include_features` directly